defs.bzl

```diff
@@ -20,15 +20,15 @@ load("//internal:ts_repositories.bzl", _ts_setup_workspace = "ts_setup_workspace
 load("//internal:build_defs.bzl", _ts_library = "ts_library")
 load("//internal:ts_config.bzl", _ts_config = "ts_config")
 load("//internal/devserver:ts_devserver.bzl", _ts_devserver = "ts_devserver_macro")
-load("//internal/karma:ts_web_test.bzl", _ts_web_test = "ts_web_test_macro")
+load("//internal/karma:ts_web_test_suite.bzl", _ts_web_test_suite = "ts_web_test_suite")
 load("//internal/protobufjs:ts_proto_library.bzl", _ts_proto_library = "ts_proto_library")
 
 ts_setup_workspace = _ts_setup_workspace
 ts_library = _ts_library
 ts_config = _ts_config
 ts_devserver = _ts_devserver
-# TODO(alexeagle): make ts_web_test work in google3
-ts_web_test = _ts_web_test
+# TODO(alexeagle): make ts_web_test_suite work in google3
+ts_web_test_suite = _ts_web_test_suite
```

Comment:
let's keep `ts_web_test` here as well so we don't break Angular

Reply:
Done